File: src/main/java/ru/openbank/meetup/demo/java/slide1/EmailDto.java
package ru.openbank.meetup.demo.java.slide1;

import lombok.Data;

import java.util.List;

/**
 * Содержит информацию, необходимую для отправки email.
 */
@Data
public class EmailDto {
    private String subject;
    private String from;
    private String personal;
    private List<String> to;
    private List<String> cc;
    private String body;
    private boolean html;
}

File: src/main/kotlin/ru/openbank/meetup/demo/kotlin/slide1/EmailDto.kt
package ru.openbank.meetup.demo.kotlin.slide1

/**
 * Содержит информацию, необходимую для отправки email.
 */
data class EmailDto(
    var subject: String? = null,
    var from: String? = null,
    var personal: String? = null,
    var to: List<String>? = null,
    var cc: List<String>? = null,
    var body: String? = null,
    var isHtml: Boolean = false
)
